Multisig/App/SceneDelegate.swift

```diff
@@ -52,6 +50,25 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
             self.scene = scene
             makeWindows(scene: scene)
         }
+
+        App.shared.appReview.startedFromNotification = connectionOptions.notificationResponse != nil
+
+        // Get URL components from the incoming user activity.
+        guard let userActivity = connectionOptions.userActivities.first,
+              userActivity.activityType == NSUserActivityTypeBrowsingWeb,
```

Comment:
all of these changes are not under feature toggle, so the app will react to the links or activities even if the toggle is off

Reply:
ok